thread.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const crypto = require('crypto');
+
+const requireOptional = (name, defaultValue) => {
+	try {
+		return require(name);
+	} catch (error) {
+		return defaultValue;
+	}
+};
```

Comment:
You don't need the optional require here as the file is not evaluated on older Node.js versions. You just need `// eslint-disable import/no-unresolved`.

Reply:
It's not about evaluation, it's about linter.

With  `// eslint-disable-line import/no-unresolved` it fails in v12 with `eslint-comments/no-unused-disable` for `import/no-unresolved`
With `// eslint-disable-line import/no-unresolved, eslint-comments/no-unused-disable` for both `eslint-comments/no-unused-disable` and `import/no-unresolved`.

Reply:
The last one is a bug of `eslint-comments/no-unused-disable` I suppose or me completely misunderstanding something.

Reply:
Oh I see. Yeah, that is annoying. You can disable `import/no-unresolved` for the whole project then, for now, in package.json.